docs/source/installation/for_dev.rst

```diff
@@ -54,7 +54,7 @@ Installing System Dependencies
 ******************************
 
 Please follow the instructions under :ref:`installing-manim` to install all
-dependencies (e.g. ``LaTeX``, ``ffmpeg``, etc.). Afterwards, proceed with the
+dependencies (e.g. ``LaTeX``, ``ffmpeg``, etc.). Afterward, proceed with the
```

Comment:
Pretty sure "afterwards" is correct.

Reply:
That’s what I thought but apparently this is grammatically correct 🧐

Reply:
Both are correct, there is no difference between the two spellings. 'Afterward' is used in the U.S. and 'afterwards' is used elsewhere. Personally, I prefer `afterwards` though I'm from the USA.

Reply:
I like to have `Afterwards`.